claat/export.go

```diff
@@ -115,11 +129,12 @@ func writeCodelab(dir string, clab *types.Codelab, ctx *types.Context) error {
 	// which will also verify output format is valid,
 	// and avoid creating empty files in case this goes wrong
 	data := &render.Context{
-		Env:      ctx.Env,
-		Prefix:   ctx.Prefix,
-		GlobalGA: ctx.MainGA,
-		Meta:     &clab.Meta,
-		Steps:    clab.Steps,
+		Env:       ctx.Env,
+		Prefix:    ctx.Prefix,
+		GlobalGA:  ctx.MainGA,
+		Meta:      &clab.Meta,
+		Steps:     clab.Steps,
+		ExtraVars: parseAdditionalVars(),
```

Comment:
`writeCodelab` is called for each codelab when multiple are given on the input, so `parseAdditionalVars` would unmarshal the same data all over again, as many time as there are codelabs.

I would do it in the main, after `flag.Parse` and then just use it here.